packages/styling/src/interfaces/ITheme.ts

```diff
@@ -2,16 +2,22 @@ import { IPalette } from './IPalette';
 import { IFontStyles } from './IFontStyles';
 import { ISemanticColors } from './ISemanticColors';
 
+export interface IThemeFlags {
```

Comment:
I don't think we need a whole list of flags for the theme. This isn't something we want to be adding to. We can have it as a sibling to isInverted, and then just document it as... internal use only? Something that cautions people that it's for a very narrow use case and most people don't need to worry about it.

Reply:
That sounds sensible to me. I've moved this onto the theme object and added a big comment ;)